### DISCUSSION TITLE
SafeLine English translation request

### DISCUSSION THREAD

**Discussion by hautph** [chaitin/SafeLine#592] [General]
Dear SafeLine support,

I am a user of SafeLine and I would like to ask for a translation of some of the English language features.

I have noticed that some features still have Chinese characters in English. For example, the "Allow/Deny list" and "IP Groups" feature still have Chinese characters in the title.

I would appreciate it if you could translate these features into English. This would make the SafeLine application more user-friendly for English-speaking users.

Thank you for your time and consideration.

**Comment by Lorna0** [comment]
Did you mean the Names of Built-in Rules and IP Groups since 4.0.0 ? Yes it's somewhat different from the previous translation and we missed it.
![image](https://github.com/chaitin/SafeLine/assets/56920727/bae091dd-84f2-49bb-b28e-488454a7f8ec)

We'll fix it later. Thanks for the feedback.

**Reply by hautph** [reply]
Yes, both "Allow/Deny list" and "IP Groups"

**Reply by Lorna0** [reply]
Released in 4.4.0，have a try.

**Comment by safe1ine** [comment]
Hi, we have recently opened a new English version, welcome to experience it

https://waf.chaitin.com/
